pkg/virtualKubelet/reflection/workload/podns.go

```diff
@@ -718,3 +719,16 @@ func (npr *NamespacedPodReflector) InferAdditionalRestarts(local, remote *corev1
 	// This should never occur, since the containers should match
 	return 0
 }
+
+// List retrieves the list of reflected pods.
+func (npr *NamespacedPodReflector) List() ([]interface{}, error) {
+	listers := map[string]virtualkubelet.Lister[*corev1.Pod]{
+		"local":  npr.localPods,
+		"remote": npr.remotePods,
```

Comment:
Is it correct to use remote pods instead of shadow pods?

Reply:
Yes, you are right. I've just noticed that the localPods informer is not related to the namespaced. Could be a good idea to implement the List method also in the PodFallbackReflector and refresh it, since it is initialized there.

Reply:
https://github.com/liqotech/liqo/pull/1886/commits/b6804eadf2953c15ea304db339bb66aa48cf487e